packages/react/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -1,215 +1,179 @@
-      borderColorDisabled,
-
-      primaryColor,
-      primaryColorHover,
-      primaryColorFocus,
-      primaryBackgroundColor,
-      primaryBackgroundColorActive,
-      primaryBackgroundColorHover,
-      primaryBackgroundColorFocus,
-      primaryBorderColor,
-
-      circularColor,
-      circularColorActive,
-      circularBackgroundColor,
-      circularBackgroundColorActive,
-      circularBackgroundColorHover,
-      circularBackgroundColorFocus,
-      circularBorderColor,
-      circularBorderColorHover,
-      circularBorderColorFocus,
-
-      textColor,
-      textColorHover,
-      textPrimaryColor,
-      textPrimaryColorHover,
-      boxShadow,
-    } = variables
-
     const { ':focus': borderFocusStyles } = getBorderFocusStyles({
       siteVariables,
-      isFromKeyboard,
-      ...(circular && {
-        borderRadius: circularBorderRadius,
-        focusOuterBorderColor: circularBorderColorFocus,
+      isFromKeyboard: p.isFromKeyboard,
+      ...(p.circular && {
+        borderRadius: v.circularBorderRadius,
+        focusOuterBorderColor: v.circularBorderColorFocus,
       }),
     })
 
     return {
-      height,
-      minWidth,
-      maxWidth,
-      color,
-      backgroundColor,
-      borderRadius,
+      height: v.height,
+      minWidth: v.minWidth,
+      maxWidth: v.maxWidth,
+      color: v.color,
+      backgroundColor: v.backgroundColor,
+      borderRadius: v.borderRadius,
       display: 'inline-flex',
       justifyContent: 'center',
       alignItems: 'center',
       position: 'relative',
-      padding: `0 ${pxToRem(paddingLeftRightValue)}`,
+      padding: v.padding,
       verticalAlign: 'middle',
       cursor: 'pointer',
 
       // rectangular button defaults
-      ...(!text && {
+      ...(!p.text && {
         outline: 0,
         borderWidth,
         borderStyle: 'solid',
-        borderColor,
-        boxShadow,
+        borderColor: v.borderColor,
+        boxShadow: v.boxShadow,
 
         ':hover': {
-          color: colorHover,
-          backgroundColor: backgroundColorHover,
-          borderColor: borderColorHover,
+          color: v.colorHover,
+          backgroundColor: v.backgroundColorHover,
+          borderColor: v.borderColorHover,
         },
 
         ':focus': {
           boxShadow: 'none',
-          ...(isFromKeyboard
+          ...(p.isFromKeyboard
             ? {
-                color: colorFocus,
-                backgroundColor: backgroundColorFocus,
+                color: v.colorFocus,
+                backgroundColor: v.backgroundColorFocus,
                 ...borderFocusStyles,
               }
-            : { ':active': { backgroundColor: backgroundColorActive } }),
+            : { ':active': { backgroundColor: v.backgroundColorActive } }),
         },
       }),
 
       // circular button defaults
-      ...(circular &&
-        !text && {
-          minWidth: height,
+      ...(p.circular &&
+        !p.text && {
+          minWidth: v.height,
```

Comment:
Hmm, shouldn't this be updated depending on the size? It seems like a special case..

Reply:
![image](https://user-images.githubusercontent.com/14183168/61869603-4975b600-aedc-11e9-9318-b7b842c8a922.png)

```tsx
      <Button size="small" circular />
      <Button circular />
```

Yep, it should

Reply:
![image](https://user-images.githubusercontent.com/14183168/61869921-167ff200-aedd-11e9-8903-d207c8efcf89.png)

Updated 👍